ubyssey/static_src/src/styles/modules/_darkmode.scss

```diff
@@ -1,16 +1,18 @@
 @import 'variables';
 
-.c-article_dark_mode {
-  float: right;
-}
-
```

Comment:
Why is this getting deleted? The class is ensuring that the dark mode toggle is to the right of the share button's bar

Reply:
I didn't think it looked very good like that

Reply:
Combining the share bar and the dark mode button makes it look a bit untidy. That is why I placed it to the right of the line.

Reply:
Lets ask editors what they prefer